source/slang/hlsl.meta.slang

```diff
@@ -2228,123 +2228,134 @@ __generic<T : __BuiltinIntegerType>
 __glsl_extension(GL_KHR_shader_subgroup_arithmetic)
 __spirv_version(1.3)
 __target_intrinsic(glsl, "subgroupAnd($0)")
-__target_intrinsic(cuda, "_waveAnd(__activemask(), $0)")
+__target_intrinsic(cuda, "_waveAnd($0)")
```

Comment:
Now that these functions are just 1-to-1 parallels to the HLSL builtin, we could simplify these `__target_intrinsic` lines in two ways:

1. Any case where you'd just write `__target_intrinsic(cuda, "someFunc($0)")` you can just write `__target_intrinsic(cuda, "someFunc")`

2. If we rename the `_waveWhatever` functions in the CUDA prelude to exactly match the HLSL names, then we can just write `__target_intrinsic(cuda)`. On a function named `WaveFoo` this should be equivalent to writing `__targert_intrinsic(cuda, "WaveFoo")` or `__target_intrinsic(cuda, "WaveFoo($0)")`.